commons/all/src/main/java/org/infinispan/commons/dataconversion/EncoderIds.java

```diff
@@ -8,7 +8,19 @@
    short IDENTITY = 1;
    short BINARY = 2;
    short UTF8 = 3;
+   /**
+    * @deprecated Since 11.0, will be removed in 14.0. Set the storage media type and use transcoding instead.
```

Comment:
Please create a sub-task on https://issues.redhat.com/browse/ISPN-11691 so that the removal isn't missed and add to the javadocs.

Reply:
@ryanemerson IMO it's too early to create those tasks, we can always run a script to find the classes/methods that were deprecated in 11.0.

Reply:
I created [ISPN-11962](https://issues.redhat.com/browse/ISPN-11962)